packages/contracts/test/utils/constants.ts

```diff
@@ -52,102 +53,101 @@ export const lensPolygonAddresses = {
 };
 /* zkevm test net is bridged to goerli */
 export const lenszkEVMAddresses = {
```

Comment:
There's a lot of datasets here that are not needed. I'd propose to remove those because it's clutter